delphi-checks/src/main/resources/org/sonar/l10n/delphi/rules/community-delphi/TooManyDefaultParameters.html

```diff
@@ -0,0 +1,20 @@
+<h2>Why is this an issue?</h2>
+<p>
+  Default parameters can be useful in cases when there is a sensible default value. They can,
+  however, damage API interfaces when used to make a method's behaviour overly configurable
+  a method with too many default parameters is confusing, unpredictable at a glance, unintuitive
+  to use, and likely overloaded in functionality.
+</p>
+<p>
+  In most cases, using method overloads, multiple methods, or configuration objects is a more
+  appropriate solution.
+</p>
+<h2>How to fix it</h2>
+<p>
+  Refactor this routine by re-evaluating the need for the default values, and either:
+</p>
+<ul>
+  <li>Removing the unnecessary default values,</li>
+  <li>Adding multiple overloads with different groups of required parameters, or</li>
+  <li>Using a configuration object approach instead</li>
```

Comment:
Do we prefer the term "configuration object" for this, or should we say something like "Grouping parameters into an object"?